### PR TITLE
fix(recipes:update): github api now requires is:pull-request parameter

### PR DESCRIPTION
**Pull request by alainsharemat**
This fix is related to issue #974.

While running `recipes:update`  I got this error:

```shell
Calculating CHANGELOG...
In CurlDownloader.php line 630:

  The "https://api.github.com/search/issues?q=95e75e5e424c5f5f611cc57073e399baaf287570" file could not be downloaded (HTTP/2 422 ):                                  
  {"message":"Query must include 'is:issue' or 'is:pull-request'","documentation_url":"https://docs.github.com/rest/search/search#search-issues-and-pull-requests"}
```

According to github's [documentation](https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-issues-and-pull-requests) : `Requests that don't include the is:issue or is:pull-request qualifier will receive an HTTP 422 Unprocessable Entity response`.

### CODE REVIEW

**Review by maxhelias — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

File: src/GithubApi.php [hunk 1]
```diff
@@ -142,7 +142,7 @@ public function getPullRequestForCommit(string $commit, string $repo): ?array
         $bestItem = null;
         foreach ($data['items'] as $item) {
             // make sure the PR referenced isn't from a different repository
-            if (false === strpos($item['html_url'], sprintf('%s/pull', $repositoryName))) {
+            if (!str_contains($item['html_url'], sprintf('%s/pull', $repositoryName))) {
```
Comment (fabpot) [inline src/GithubApi.php]: Can you revert the 2 unrelated changes?

**Review by fabpot — CHANGES_REQUESTED** [review]
Should also be rebased on 1.x.

File: src/GithubApi.php (hunk 1 above)
Comment (alainsharemat) [reply src/GithubApi.php]: I did. Had to commit this code to make fabbot.io happy in the first place.

**Comment by fabpot** [comment]
Thank you @alainsharemat.
